### PR TITLE
Add xz-git package.

### PR DESCRIPTION
**Pull request by fracting**
Tested with Travis CI: https://travis-ci.org/fracting/xz/jobs/73616615#L3205

Note that I didn't test 64bit version. Also I didn't test git fetch, I use source code fetch by host git in Travis CI.

Thanks for review!

### CODE REVIEW

**Comment by Alexpux** [comment]
You need add in PKGBUILD's that build programs from any git/cvs/svn/bzr sources entries ````provides```` and ````conflicts````. See our PKGBUILD's for git sources

**Comment by fracting** [comment]
Thanks for review, I updated the patch and rebuild again: https://travis-ci.org/fracting/xz/jobs/73650598

Could you review again? Thanks!

**Comment by Alexpux** [comment]
You need rebase your changes to last our commit. In other case history is not clear.

**Comment by fracting** [comment]
Thanks Alexpux, I updated my branch, could you review again? Thanks!

File: mingw-w64-xz-git/PKGBUILD [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+depends=("${MINGW_PACKAGE_PREFIX}-gcc-libs" "${MINGW_PACKAGE_PREFIX}-gettext")
+provides=("${MINGW_PACKAGE_PREFIX}-${_realname}")
+conflicts=("${MINGW_PACKAGE_PREFIX}-${_realname}")
+options=('staticlibs' 'strip')
+source=("git+http://tukaani.org/xz.git")
+sha256sums=('SKIP')
+
+pkgver() {
+  cd ${srcdir}/${_realname}
+  printf "%s.%s.%s" "$_ver_base" "$(git rev-list --count HEAD)" "$(git rev-parse --short HEAD)"
```
Comment (Alexpux) [inline mingw-w64-xz-git/PKGBUILD]: does git repo have ````tags```` like we do here: https://github.com/Alexpux/MINGW-packages/blob/master/mingw-w64-codelite-git/PKGBUILD#L38 ? Then you don't need manually maintain ````_ver_base````

Comment (fracting) [inline mingw-w64-xz-git/PKGBUILD]: Hi Alexpux, thanks for review, good advice, I didn't think about that before, but after checking I found xz git repo doesn't have tags.

**Pull request merged**